packages/babel-helper-create-class-features-plugin/src/decorators.ts

```diff
@@ -919,18 +919,11 @@ function createPrivateBrandCheckClosure(brandName: t.PrivateName) {
 }
 
 function usesPrivateField(expression: t.Node) {
-  try {
-    t.traverseFast(expression, node => {
-      if (t.isPrivateName(node)) {
-        // TODO: Add early return support to t.traverseFast
-        // eslint-disable-next-line @typescript-eslint/only-throw-error
-        throw null;
-      }
-    });
-    return false;
-  } catch {
-    return true;
-  }
+  return t.traverseFast(expression, node => {
+    if (t.isPrivateName(node)) {
+      return "stop";
```

Comment:
Is this guaranteed to use a new-enough version of babel types?

Reply:
Oops, this is using `@babel/core`.